flax/linen/module.py

```diff
@@ -1256,7 +1236,7 @@ def __post_init__(self) -> None:
           and self.parent._state.setup_called == SetupState.TRANSFORMED
       )
       # Perform name-collision check.
-      if self.parent._name_taken(self.name, self, reuse_scopes=reuse_scopes):
+      if self.parent._name_taken(self.name, reuse_scopes=reuse_scopes):
```

Comment:
Can you comment about this change, seems a little unrelated. Also, not sure why but currently the `module` argument is not being used inside `_name_taken`.

Reply:
Yes, it's an unrelated change, removed that argument since it's unused. Happy to undo.